### DISCUSSION TITLE
Projects[beta] details get stuck at loading when user is not logged in

### DISCUSSION THREAD

**Discussion by aybehrouz** [community/community#20246] [Projects and Issues]
When a user, using **Firefox** or **Edge**, tries to check the project details of a public project created by projects[beta] the README gets stuck at loading:

![image](https://user-images.githubusercontent.com/75391306/178116880-6febf37c-87b6-4a0e-b046-9ef2a54b335e.png)

If I log into my account the problem is solved:

![image](https://user-images.githubusercontent.com/75391306/178116989-965683b3-a81a-4377-975a-376c1e597941.png)

This is the link of the project: https://github.com/orgs/Argennon-Project/projects/4/views/1

I checked other public projects and it seems that the same issue exists. For example this project:
https://github.com/orgs/microsoft/projects/240/views/1

**Comment by billgeo** [comment]
I'm also having this issue when viewing anonymously. Example: https://github.com/orgs/linz/projects/12/views/3
